deploying-daml/daml-on-fabric/daml_create_app.md

```diff
@@ -0,0 +1,5 @@
+First thing we'll need to do is get our DAML application. In this case we'll be using an example application that's also used in our getting started guide. It's called `create-daml-app` because, well, you can use it as a base for creating DAML apps. Brilliant!
+
+```
+daml create-daml-app my-app
+```{{execute T1}}
```

Comment:
Maybe think about removing this step and create the project in the background. Ppl will have their own apps anyway and the process roughly starts once they wanna deploy the `.dar` file

Reply:
I'd prefer to keep it for the example. I see your point but I wouldn't be surprised if many people trying this scenario haven't written any DAML code.